<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Nova's Song Organizer</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../../../../../nova logo.ico</normaloff>../../../../../../nova logo.ico</iconset>
  </property>
  <widget class="QPushButton" name="folderButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>111</width>
     <height>32</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Choose Folder ...</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>781</width>
     <height>561</height>
    </rect>
   </property>
   <property name="sortingEnabled">
    <bool>true</bool>
   </property>
   <attribute name="verticalHeaderShowSortIndicator" stdset="0">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QLineEdit" name="searchBar">
   <property name="geometry">
    <rect>
     <x>331</x>
     <y>4</y>
     <width>355</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="searchButton">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>-1</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Search</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>126</x>
     <y>1</y>
     <width>28</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../../../../../../nova logo.ico</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>158</x>
     <y>6</y>
     <width>174</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>novanewchorus@gmail.com</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
